meta/run.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+print_help() {
+    NAME=$(basename "$0")
+    cat <<EOF
+Usage: $NAME [arguments...] [arguments to Jakt...]
+Build and run a Jakt program!
+Optional arguments:
+    --build-compiler: Build the Jakt compiler before compiling the program (equivalent to running cargo build first).
+    --help: Print this message.
+EOF
+}
+
+if [ $# -eq 0 ]; then
+    print_help
+    exit 1
+fi
+
+BUILD_JAKT=0
+
+for argument in "$@"
+do
+    if [ $argument = "--help" ]; then
+        print_help
+        exit
+    fi
+    if [ $argument = "--build-compiler" ]; then
+        BUILD_JAKT=1
+        shift
+    fi
+done
+
+# Check to make sure clang is installed, since we need it to run the program.
+if ! command -v clang++ &> /dev/null
+then
+    echo "clang++ was not found on your PATH, and it's required for building the output C++ program. Install it, then come back here :^)"
+    exit 1
+fi
+
+if [ $BUILD_JAKT = 1 ]; then
+    cargo build
+fi
+
+# Assuming that jakt is in the normal place after it's built...
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+$SCRIPT_DIR/../target/debug/jakt "$@"
+
+JAKT_EXIT=$?
+if [ $JAKT_EXIT != 0 ]; then
+    echo "Compilation failed :^("
+    exit $JAKT_EXIT
+fi
+
+clang++ -std=c++20 -Iruntime -Wno-user-defined-literals -o jakt.out output.cpp 
+./jakt.out    
```

Comment:
No trailing whitespace, but trailing newline please

Reply:
Fixed, along with adding a note about it to README.md. Sorry I completely spaced on documentation, I wasn't 100% awake when I was doing this :)